src/esbmc/bmc.cpp

```diff
@@ -226,84 +181,31 @@ smt_convt::resultt bmct::run_decision_procedure(
   str << "\nRuntime decision procedure: ";
   output_time(sat_stop - sat_start, str);
   str << "s";
-  status(str.str());
+  msg.status(str.str());
 
   return dec_result;
 }
 
 void bmct::report_success()
 {
-  status("\nVERIFICATION SUCCESSFUL");
-
-  switch(ui)
-  {
-  case ui_message_handlert::OLD_GUI:
-    std::cout << "SUCCESS" << std::endl
-              << "Verification successful" << std::endl
-              << "" << std::endl
-              << "" << std::endl
-              << "" << std::endl
-              << "" << std::endl;
-    break;
-
-  case ui_message_handlert::GRAPHML:
-    break;
-
-  case ui_message_handlert::PLAIN:
-    break;
-
-  case ui_message_handlert::XML_UI:
-  {
-    xmlt xml("cprover-status");
-    xml.data = "SUCCESS";
-    std::cout << xml;
-    std::cout << std::endl;
-  }
-  break;
-
-  default:
-    assert(false);
-  }
+  msg.status("\nVERIFICATION SUCCESSFUL");
```

Comment:
Shall we add `\n` after `SUCCESSFUL`?

Reply:
Every `status` adds a `\n`. The original statement was "\nVERIFICATION SUCCESSFUL\n"